app/main/views/services.py

```diff
@@ -3,10 +3,10 @@
 from flask import url_for as base_url_for
 from sqlalchemy.exc import IntegrityError, DatabaseError
 
-from . import main
-from .. import db
-from ..models import ArchivedService, Service, Supplier
-from ..validation import validate_json_or_400, validate_updater_json_or_400
+from app.main import main
```

Comment:
I'd keep the relative imports here since that's what we're using in `views/suppliers.py` and almost everywhere else